### PR TITLE
Update promotion to target .NET 9 Eng

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -216,43 +216,16 @@ extends:
...
-                  -azdoOrg "dnceng"
-                  -azdoProject "internal"
-                  -barToken $(MaestroAccessToken)
-                  -githubPAT $(BotAccount-dotnet-bot-repo-PAT)
+    - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/release/9.0') }}:
       - stage: Promote_Arcade_To_Latest
-        displayName: Promote Arcade to '.NET Eng - Latest' channel
+        displayName: Promote Arcade to '.NET 9 Eng' channel
         dependsOn:
           - Validate_Publishing
```
Comment (riarenas) [inline azure-pipelines.yml:223]: This dependsOn should also go

Comment (mmitche) [reply azure-pipelines.yml:223]: doh

**Comment by riarenas** [comment]
I think you might have forgotten to push the latest commit that removes the dependsOn?

**Comment by mmitche** [comment]
Somehow, yes. Man I am off today.

**Pull request merged**